src/CosmosDB/CosmosDB.Test/CosmosDB.Test.csproj

```diff
@@ -4,7 +4,8 @@
   </PropertyGroup>
   <Import Project="$(MSBuildThisFileDirectory)..\..\Az.Test.props" />
   <ItemGroup>
+    <PackageReference Include="Microsoft.Azure.Management.CosmosDB" Version="3.5.0-preview" />
     <PackageReference Include="Microsoft.Azure.Management.Network" Version="21.0.0" />
-    <PackageReference Include="Microsoft.Azure.Management.CosmosDB" Version="3.3.0" />
+    <PackageReference Include="Microsoft.Azure.KeyVault" Version="3.0.5" />
```

Comment:
The version should be 3.0.1, please refer to https://github.com/Azure/azure-powershell/blob/4bdd4a117c2d8a37f08db58a65a31fe119afb3b1/src/KeyVault/KeyVault/KeyVault.csproj#L18

Reply:
directly referring proj